### PR TITLE
Variables intro translation

### PR DESCRIPTION
**Pull request by pratik98**
Please review the changes and let me know if the translation is on the lines of what is expected. Then i will proceed further with next pages.

### CODE REVIEW

File: variables_json/PARSED_CONTENT/PROPERTIES_FILES/variables/Variables_json_variables-intro_hi.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
-    "variables-intro_key1": "<span style=\"\">Variables</span> क्या हैं?",
-    "variables-intro_key2": "<b><span style=\"\">VARIABLE</span></b> एक मेमोरी/लोकेशन है जिसमें डेटा स्टोर किया जाता है। जिसका मूल्य हम बदल सकते हैं उसे <span style=\"\">VARIABLE</span> कहा जाता है। आप VkFSSUFCTEU= को कोई भी नाम दे सकते हैं और <b><span style=\"\">variable</span></b> <span style=\"\"></span> कुछ भी स्टोर कर सकते हैं।",
-    "variables-intro_key3": "जिस तरह हम <span style=\"color: #eb9371; background-color: rgba(39, 41, 43, 0.83); box-shadow: 2px 2px 2px rgba(22, 9, 1, 0.6); border-radius: 2px; padding: 2px\">123</span> को <span style=\"color: #eb9371; background-color: rgba(39, 41, 43, 0.83); box-shadow: 2px 2px 2px rgba(22, 9, 1, 0.6); border-radius: 2px; padding: 2px\">variable_x</span> नाम के VARIABLE में स्टोर कर सकते हैं।",
+    "variables-intro_key1": "<span style=\"\">वेरिएबलस</span> क्या हैं?",
```
Comment (anandpatel504) [inline variables_json/PARSED_CONTENT/PROPERTIES_FILES/variables/Variables_json_variables-intro_hi.json:2]: Variable is a keyword it should be in English

File: variables_json/PARSED_CONTENT/PROPERTIES_FILES/variables/Variables_json_variables-intro_hi.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+    "variables-intro_key2": "<b><span style=\"\">वेरिएबल</span></b> एक मेमोरी/लोकेशन है जिसमें डेटा स्टोर किया जाता है। जिसका मूल्य हम बदल सकते हैं उसे <span style=\"\">वेरिएबल</span> कहा जाता है। आप <b><span style=\"\">वेरिएबल</span></b> को कोई भी नाम दे सकते हैं और <b><span style=\"\">वेरिएबल</span></b> मे  कुछ भी स्टोर कर सकते हैं।",
+    "variables-intro_key3": "जिस तरह हम <span style=\"color: #eb9371; background-color: rgba(39, 41, 43, 0.83); box-shadow: 2px 2px 2px rgba(22, 9, 1, 0.6); border-radius: 2px; padding: 2px\">123</span> को <span style=\"color: #eb9371; background-color: rgba(39, 41, 43, 0.83); box-shadow: 2px 2px 2px rgba(22, 9, 1, 0.6); border-radius: 2px; padding: 2px\">variable_x</span> नाम के वेरिएबल में स्टोर कर सकते हैं।",
```
Comment (anandpatel504) [inline variables_json/PARSED_CONTENT/PROPERTIES_FILES/variables/Variables_json_variables-intro_hi.json:4]: same here (keywords will be not change)

File: variables_json/PARSED_CONTENT/PROPERTIES_FILES/variables/Variables_json_variables-intro_hi.json [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+    "variables-intro_key7": "जैसा कि आपको याद है कि <span style=\"color: #eb9371; background-color: rgba(39, 41, 43, 0.83); box-shadow: 2px 2px 2px rgba(22, 9, 1, 0.6); border-radius: 2px; padding: 2px\">9.9</span> <span style=\"\">float</span> है। इसका मतलब है कि हम <span style=\"\">वेरिएबल</span> को अलग-अलग वैल्यू दे सकते हैं। <span style=\"color: #eb9371; background-color: rgba(39, 41, 43, 0.83); box-shadow: 2px 2px 2px rgba(22, 9, 1, 0.6); border-radius: 2px; padding: 2px\">a</span> की वैल्यू या मूल्य नहीं बदला गया है क्योंकि हमने इसे अपडेट या इसमे बदलाव नहीं किया है। अब हम <span style=\"color: #eb9371; background-color: rgba(39, 41, 43, 0.83); box-shadow: 2px 2px 2px rgba(22, 9, 1, 0.6); border-radius: 2px; padding: 2px\">a</span> को अपडेट कर सकते हैं और एक नया <span style=\"\">वेरिएबल</span> <span style=\"color: #eb9371; background-color: rgba(39, 41, 43, 0.83); box-shadow: 2px 2px 2px rgba(22, 9, 1, 0.6); border-radius: 2px; padding: 2px\">b</span> बना सकते हैं।",
```
Comment (anandpatel504) [inline variables_json/PARSED_CONTENT/PROPERTIES_FILES/variables/Variables_json_variables-intro_hi.json:8]: same here, need to change keyword (variable)

**Comment by pratik98** [comment]
Updated Variable keyword

**Pull request merged**